tests/shared/features/shared/session_steps.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Browser session steps
+
+  Background: Access some content in the main session
+    Given I go to "/static_pages/tab_1"
+    Then I should see "First browser tab"
+
+  Scenario: Browser sessions do not interfere with each other
```

Comment:
Please add new minimal static pages for this test. It's irritating to read about browser tabs when they do not matter at all